### PR TITLE
(v2.11) Fixes enums and docBlock -> doc_block

### PR DESCRIPTION
**Pull request by kevincobain2000**

### CODE REVIEW

File: src/LaravelRequestDocsMiddleware.php [hunk 1]
```diff
@@ -28,9 +28,15 @@ class LaravelRequestDocsMiddleware extends QueryLogger
     public function handle(Request $request, Closure $next): Response
     {
         if (!$request->headers->has('X-Request-LRD') || !config('app.debug')) {
-            return $next($request);
+            $response    = $next($request);
+            $jsonContent = json_encode([
+                'data' => $response->getData()
+            ]);
+            $response->setContent($jsonContent);
+            return $response;
```
Comment (kevincobain2000) [inline src/LaravelRequestDocsMiddleware.php:36]: Note:

```
There was 1 failure:

1) Rakutentech\LaravelRequestDocs\Tests\LaravelRequestDocsMiddlewareTest::testMissingLRDHeader
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 '{\n
-    "test": true\n
+    "data": {\n
+        "test": true\n
+    }\n
 }'
```

Also note: https://github.com/rakutentech/laravel-request-docs/pull/177/files#r1124390042

Comment (kitloong) [reply src/LaravelRequestDocsMiddleware.php:36]: Replied at https://github.com/rakutentech/laravel-request-docs/pull/177/files#r1124440150

Comment (kevincobain2000) [reply src/LaravelRequestDocsMiddleware.php:36]: Actually this is a good test.
We don't want accidentally create a security risk.

**Comment by kevincobain2000** [comment]
@kitloong 
Please re-review
Please merge all and push the new tag and create the new release
No need for a meeting this Monday

**Pull request merged**